test/media_sources/facebook_media_source_test.rb

```diff
@@ -21,4 +21,40 @@ def setup; end
       assert_not_nil(post)
     end
   end
+
+  test "extracted post has images and videos uploaded to S3" do
```

Comment:
This (and all other) S3 tests should only run if AWS uploading is actually configured. As-is, these tests fail if Hypatia is configured without AWS creds.